reference/CloudPay-API-Specification.yaml

```diff
@@ -1375,12 +1375,177 @@ paths:
+                    minLength: 1
+                  CustomFields:
+                    type: array
+                    uniqueItems: true
+                    minItems: 1
+                    items:
+                      required:
+                        - Id
+                        - Name
+                        - Label
+                        - Value
+                      properties:
+                        Id:
+                          type: string
+                          minLength: 1
+                        Name:
+                          type: string
+                          minLength: 1
+                        Label:
+                          type: string
+                          minLength: 1
+                        Value:
+                          type: string
+                          minLength: 1
+                required:
+                  - EmailAddress
+                  - FirstName
+                  - LastName
+                  - CustomFields
+                x-examples:
+                  example-1:
+                    EmailAddress: customer@email.com
+                    FirstName: Lorem
+                    LastName: Ipsum
+                    CustomFields:
+                      - Id: 2b31e959-c540-4d39-ba76-5e6ef59df772
+                        Name: Date of Birth
+                        Label: Date of Birth
+                        Value: '1988-08-19'
+                      - Id: db3fb17a-2b10-481e-b9fc-6fc4303ce280
+                        Name: Favourite Rugby Team
+                        Label: Favourite Rugby Team
+                        Value: Hull FC
+              examples:
+                Successful Update Response:
+                  value:
+                    EmailAddress: customer@email.com
+                    FirstName: Lorem
+                    LastName: Ipsum
+                    CustomFields:
+                      - Id: 2b31e959-c540-4d39-ba76-5e6ef59df772
+                        Name: Date of Birth
+                        Label: Date of Birth
+                        Value: '1988-08-19'
+                      - Id: db3fb17a-2b10-481e-b9fc-6fc4303ce280
+                        Name: Favourite Rugby Team
+                        Label: Favourite Rugby Team
+                        Value: Hull FC
+        '400':
+          description: Bad Request
+        '403':
+          description: Forbidden
+        '500':
+          description: Error Response
+      description: |-
+        ![In Review](https://img.shields.io/badge/Status-In%20Review-green?style=flat-square)
+
+        The update Account endpoint allows a user to update their user details.
+
+        Please be aware that the users email can not be updated via the API.
+
+
+        **Note** *Only one of the four authorisation parameters are required*
+      requestBody:
+        content:
+          application/json:
+            schema:
+              description: ''
+              type: object
+              properties:
```

Comment:
this looks like a candidate to be extracted to its own schema potentially? used in response and request body